src/helpers/agGrid.ts

```diff
@@ -554,8 +561,17 @@ export const executionToAGGridAdapter = (
         data as Execution.DataValue[][],
         headerItems[0],
     );
+
     const rowData = minimalRowData.map((dataRow: Execution.DataValue[], dataRowIndex: number) =>
-        getRow(dataRow, dataRowIndex, columnFields, rowHeaders, headerItems[0], intl),
+        getRow(
+            dataRow,
+            dataRowIndex,
+            columnFields,
+            rowHeaders,
+            headerItems[0],
+            getSubtotalStyles(resultSpec.dimensions ? resultSpec.dimensions[0] : null),
```

Comment:
Shouldn't the fallback parameter value be `undefined`? `getSubtotalsStyles()` expects `AFM.IDimension` or nothing.